usr/local/www/interfaces_vlan_edit.php

```diff
@@ -80,7 +80,7 @@
 
 	do_input_validation($_POST, $reqdfields, $reqdfieldsn, $input_errors);
 
-	if ($_POST['tag'] && (!is_numericint($_POST['tag']) || ($_POST['tag'] < '1') || ($_POST['tag'] > '4094'))) {
+	if (!is_numericint($_POST['tag']) || ($_POST['tag'] < '1') || ($_POST['tag'] > '4094')) {
```

Comment:
yeah, but keep a isset() call is good, and it doesn't depend of the value. pfSense PHP needed to be patched when we moved to 5.5 because of there are too many places where a non-defined variable or array key is accessed. On recent PHP versions it throws an error, what we changed to a notice. We need to remove all those access without checks to be able to remove PHP patch.